cmake/SetupCharm.cmake

```diff
@@ -28,7 +28,7 @@ endif(NOT TARGET Charmxx)
 #       with GCC
 string(
     REGEX REPLACE "<CMAKE_CXX_COMPILER>"
-    "${CHARM_COMPILER} -pthread -module CommonLBs -no-charmrun"
+    "${CHARM_COMPILER} -pthread -module EveryLB -no-charmrun"
```

Comment:
Does this change the way Charm++ needs to be built in our installation instructions?